livekit/attrs.go

```diff
@@ -0,0 +1,19 @@
+package livekit
+
+// Names of participant attributes for SIP.
+const (
+	// AttrSIPPrefix is shared for all SIP attributes.
+	AttrSIPPrefix = "lk.sip."
+	// AttrSIPCallID attribute contains LiveKit SIP call ID.
+	AttrSIPCallID = AttrSIPPrefix + "callID"
+	// AttrSIPTrunkID attribute contains LiveKit SIP Trunk ID used for the call.
+	AttrSIPTrunkID = AttrSIPPrefix + "trunkID"
+	// AttrSIPDispatchRuleID attribute contains LiveKit SIP DispatchRule ID used for the inbound call.
+	AttrSIPDispatchRuleID = AttrSIPPrefix + "ruleID"
+	// AttrSIPFromNumber attribute contains number from which the call was made.
+	// This attribute will be omitted if HidePhoneNumber is set.
+	AttrSIPFromNumber = AttrSIPPrefix + "fromNumber"
```

Comment:
I think the `fromNumber` and `toNumber` is a bit confusing because it would flip depending on who dialed.

I would vote for `sip.phoneNumber` and `sip.trunkPhoneNumber` to differentiate.

Reply:
You are right, it may be confusing. Users could know the direction by checking if Dispatch Rule ID is set (inbound only), but better make it obvious :+1:

Reply:
Done